src/network/connection.h

```diff
@@ -705,7 +705,7 @@ class Connection
 	~Connection();
 
 	/* Interface */
-	ConnectionEventPtr waitEvent(u32 timeout_ms);
+	ConnectionEventPtr waitEvent(u32 timeout_us);
```

Comment:
connection.cpp L1439 uses `timeout_ms` but this function uses `timeout_us`. One of them is incorrect or misleading.

Reply:
Oops, good catch. This is just a typo.
(I tried improving the accuracy to μs, but moved that to another branch. Seems like this is a leftover from that.)